numba/npyufunc/parallel.py

```diff
@@ -230,7 +230,21 @@ def build_gufunc_wrapper(py_func, cres, sin, sout, cache):
 
 
 _backend_init_thread_lock = threadRLock()
-_backend_init_process_lock = procRLock()
+try:
+    _backend_init_process_lock = procRLock()
+except OSError as e:
+    # probably lack of /dev/shm for semaphore writes, warn the user
```

Comment:
should this comment be part of the message since it suggests a possible reason for the user to look up?